<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuxiliaryValveCommandWindow</class>
 <widget class="QWidget" name="AuxiliaryValveCommandWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="12">
    <widget class="QSlider" name="valve12VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QSlider" name="valve6VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSlider" name="valve3VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>10</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSlider" name="valve0VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="13">
    <widget class="QSlider" name="valve13VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="14">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>14</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="14">
    <widget class="QSlider" name="valve14VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="13">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>13</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>9</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>11</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>12</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSlider" name="valve5VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="15">
    <widget class="QSlider" name="valve15VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QSlider" name="valve7VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="valve1VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QSlider" name="valve9VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSlider" name="valve4VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QSlider" name="valve10VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="15">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>15</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QSlider" name="valve8VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="11">
    <widget class="QSlider" name="valve11VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSlider" name="valve2VerticalSlider">
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="enableValve0CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="enableValve1CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="enableValve2CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="enableValve3CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="enableValve4CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QCheckBox" name="enableValve5CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QCheckBox" name="enableValve6CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QCheckBox" name="enableValve7CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QCheckBox" name="enableValve8CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QCheckBox" name="enableValve9CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="10">
    <widget class="QCheckBox" name="enableValve10CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="11">
    <widget class="QCheckBox" name="enableValve11CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="12">
    <widget class="QCheckBox" name="enableValve12CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="13">
    <widget class="QCheckBox" name="enableValve13CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="14">
    <widget class="QCheckBox" name="enableValve14CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="15">
    <widget class="QCheckBox" name="enableValve15CheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
